### PR TITLE
reduce window of characters after the mention marker character for keeping the search list open

### PR DESCRIPTION
**Pull request by b0b3rt**
Many users (including me) have found it quite annoying that the search element for mentions stays open for an entire paragraph (until the next line break).  This PR caps it to 20 characters, which is long enough that you can reasonably search for longer post titles, but not so long that it should be staying open for more than the length of a sentence.  There's probably some better heuristics but this is a pretty easy 80:20.

If the EA forum wants this forum-gated... I'm not actually sure what the best way to accomplish that is... maybe extending the mention plugin configuration and passing a value in through that?  But let me know.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205245040010276) by [Unito](https://www.unito.io)

### CODE REVIEW

File: public/lesswrong-editor/src/ckeditor5-mention/src/mentionui.js [hunk 1]
```diff
@@ -730,7 +730,8 @@ function getLastValidMarkerInText( feedsWithPattern, text ) {
 // @param {Number} minimumCharacters
 // @returns {RegExp}
 export function createRegExp( marker, minimumCharacters ) {
-  const numberOfCharacters = minimumCharacters == 0 ? '*' : `{${ minimumCharacters },}`;
+	// End after 20 characters, since it's horribly annoying otherwise.
+  const numberOfCharacters = minimumCharacters == 0 ? '*' : `{${ minimumCharacters },20}`;
```
Comment (jpaddison3) [inline public/lesswrong-editor/src/ckeditor5-mention/src/mentionui.js:734]:
```suggestion
  const numberOfCharacters = minimumCharacters === 0 ? '*' : `{${ minimumCharacters },20}`;
```

Comment (jpaddison3) [reply public/lesswrong-editor/src/ckeditor5-mention/src/mentionui.js:734]: Why is it that minimum being 0 removes the max?

**Review by jpaddison3 — APPROVED** [review]
ASTLAC

File: public/lesswrong-editor/src/ckeditor5-mention/src/mentionui.js (hunk 1 above)
Comment (b0b3rt) [reply public/lesswrong-editor/src/ckeditor5-mention/src/mentionui.js:734]: Mostly an oversight - in practice both of our feeds have `minimumCharacters: 1`, but I agree that should be changed

**Pull request merged**